CHANGELOG.md

```diff
@@ -8,9 +8,32 @@
 * For the time being, MapboxCoreNavigation.framework depends on a build of MapboxNavigationNative.framework that is only available to authorized beta testers. Please contact your Mapbox sales representative or [support team](https://support.mapbox.com/) to learn more about the beta testing program. ([#2412](https://github.com/mapbox/mapbox-navigation-ios/pull/2412))
 * MapboxNavigation depends on Mapbox Maps SDK for iOS v6.0.0. Before CocoaPods or Carthage can download Mapbox.framework, you need to create a special-purpose access token. See [the updated installation instructions in the readme](./README.md#installing-the-latest-prerelease) for more details. ([#2437](https://github.com/mapbox/mapbox-navigation-ios/pull/2437)) 
 * Xcode 11.4.1 or above is now required for building this SDK from source. ([#2417](https://github.com/mapbox/mapbox-navigation-ios/pull/2417))
+
+### Dependencies updates
+
+* Updated `MapboxNavigationNative` version to `xx.yy.zz` ([]())
+* Updated `MapboxDirections` version to `x.y.z`. ([]())
+* Updated `Turf.swift` version to `0.5.0`. `CoordinateBounds` was replaced by `Turf.BoundingBox`, `Polyline` replaced by `LineString`, `MGLFeature` initializers refined as well as various bugs fixed in that version ([#2393](https://github.com/mapbox/mapbox-navigation-ios/pull/2393))
+
+### UI imporvements
+
+* Added Junction Guidance Views support ([#2393](https://github.com/mapbox/mapbox-navigation-ios/pull/2393))
```

Comment:
We’re maintaining a running changelog in this repository, so as long as everyone includes relevant changelog entries in individual PRs, I don’t think it’s necessary to update the changelog to reflect the final state of the release. We wouldn’t be able to land any of the changes in this PR until everything else lands, which would be unfortunate.